src/main/java/com/chopshop166/chopshoplib/drive/MockSwerveModule.java

```diff
@@ -0,0 +1,60 @@
+package com.chopshop166.chopshoplib.drive;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.kinematics.SwerveModuleState;
+import edu.wpi.first.util.sendable.SendableBuilder;
+
+public class MockSwerveModule implements SwerveModule {
+
+    private Translation2d location = new Translation2d();

```

Comment:
Why is this getting a default value? Just set it from the constructor. Also make it final.

Reply:
I I made this final and now it is declared in the constructor